### PR TITLE
Add build settings for tvOS

### PR DESCRIPTION
**Pull request by brentleyjones**

### CODE REVIEW

File: UniversalFramework_Base.xcconfig [hunk 1]
```diff
@@ -6,16 +6,20 @@
...
-LD_RUNPATH_SEARCH_PATHS[sdk=iphoneos*]        = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=iphonesimulator*] = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=watchos*]         = $(inherited) '@executable_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=watchsimulator*]  = $(inherited) '@executable_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]           = $(inherited) '@executable_path/../Frameworks' '@loader_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=iphoneos*]         = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=iphonesimulator*]  = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=watchos*]          = $(inherited) '@executable_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=watchsimulator*]   = $(inherited) '@executable_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=appletvos*]        = $(inherited) '@executable_path/Frameworks'
```
Comment (segiddins) [inline UniversalFramework_Base.xcconfig]: what about `@loader_path`?

Comment (brentleyjones) [inline UniversalFramework_Base.xcconfig]: Lots of this is really new to me :smile:. Why wasn't `@loader_path` needed for watchOS?

Comment (brentleyjones) [inline UniversalFramework_Base.xcconfig]: It seems like it should have. Amending.

**Comment by mrackwitz** [comment]
Thanks a lot! :tada:

**Comment by mrackwitz** [comment]
As that could be confirmed to be working on actual hardware, this can be merged.

**Pull request merged**